library/aesni.c

```diff
@@ -42,6 +42,11 @@
 
 #if defined(MBEDTLS_HAVE_X86_64)
 
+#if defined(_MSC_VER) && defined(_M_X64)
+#define MBEDTLS_HAVE_MSVC_X64_INTRINSICS
```

Comment:
Could be nice to have this define in config already and then reuse it. The check defined(_MSC_VER) && defined(_M_X64) comes up quite often

Reply:
We agreed (with MartinS) to minimize/avoid changes to config.h. As for addressing your comment, instead of introducing another config option, I cleaned up the code and so after I removed some of "defined(_MSC_VER) && defined(_M_X64)" there's less of them.